src/olympia/migrations/939-drop-l10n-from-bio-user-profile-cleanup.sql

```diff
@@ -0,0 +1,36 @@
+/* Follow-on from migration 938 -
+Drop the (now obsolete) bio and old translations. */
+
+DROP PROCEDURE IF EXISTS drop_bio_fk;
+
+DELIMITER ';;'
+
+CREATE PROCEDURE drop_bio_fk() BEGIN
+    /* drop the bio foreign key so we can clear the translations.
+    - we know prod and local dbs have different keys, so try to handle both */
+
+    /* prod fk name */
+    IF EXISTS (SELECT 1 FROM information_schema.table_constraints WHERE table_schema = (SELECT DATABASE()) AND table_name = 'users' AND constraint_name = 'users_ibfk_1') THEN
+        ALTER TABLE reviews DROP FOREIGN KEY users_ibfk_1;
```

Comment:
Wrong copy paste, you have the right `table_name` in the `SELECT` but the wrong one in the `ALTER`